src/main/docker/scripts/export-file-based-secrets.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# Copyright 2017-2021 Micro Focus or one of its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Create a convenience function for info logging.
+info() {
+    echo "[$(date +%H:%M:%S.%3NZ) #$(printf '%03X\n' $$).??? INFO  -            -   ] ${0##*/}: $@"
+}
+
+# Create a convenience function for error logging.
+error() {
+    echo "[$(date +%H:%M:%S.%3NZ) #$(printf '%03X\n' $$).??? ERROR  -            -   ] ${0##*/}: $@"
+}
```

Comment:
Just use `echo` instead of these functions, and then when calling the script pipe it through the `caf-log-format.sh` script.  That way we'll keep the logging format encapsulated.